### PR TITLE
Update reqs, fix TF bug

### LINKED ISSUE #3: update code requirements, fix train.py
> From https://github.com/iterative/dvc.org/pull/762#issuecomment-548222456

Req's are pretty old:

https://github.com/iterative/example-versioning/blob/ac7473d8fb9453fa3f9634941d6aa7ac40979a5f/requirements.txt#L1-L3

~~(but work. Last tested on Python 3.7)~~

UPDATE: Actually it's `tensorflow==1.13.1` now, see https://github.com/iterative/example-versioning/issues/3#issuecomment-548495290.

---

Update https://dvc.org/doc/tutorials/versioning accordingly if this code changes.

### PR DESCRIPTION
**Pull request by casperdcl**
fixes #3 

- [x] update to `tensorflow==2`
- [x] add missing `scipy` dependency
- [x] fix metrics by using `CSVLogger`
- [x] neaten progress with `tqdm`
- [x] soft-pin dependencies on major version
- [x] correct `README.md` link to documentation

## TODO
After merge, update https://dvc.org/doc/tutorials/versioning to use `metrics.csv` rather than `metrics.json`

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-tensorflow==1.13.1
-keras==2.2.4
-pillow==5.3.0
-
+pillow>=5.3,<6
+scipy
+tensorflow==2
+tqdm==4
```
Comment (jorgeorpinel) [inline requirements.txt]: I'm getting

```console
$ pip install -r requirements.txt
ERROR: Could not find a version that satisfies the requirement tqdm==4 (from -r requirements.txt (line 4)) (from versions: 1.0, ... 4.37.0)
ERROR: No matching distribution found for tqdm==4 (from -r requirements.txt (line 4))
```

Comment (casperdcl) [reply requirements.txt]: ergh silly mistake

Comment (jorgeorpinel) [reply requirements.txt]: Happens. Worksnow. Trying train.py...

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # Example: Versioning 
 
-Datasets and ML model versioning example
-[for Get Started](https://dvc.org/doc/get-started/example-versioning).
+Datasets and ML model getting started
+[versioning tutorial](https://dvc.org/doc/tutorials/versioning).
```
Comment (jorgeorpinel) [inline README.md]: 👍

**Comment by jorgeorpinel** [comment]
Just pending:

- [ ] after merge, update https://dvc.org/doc/tutorials/versioning to use `metrics.csv` rather than `metrics.json`

I'll address this myself 🙂

**Comment by shcheklein** [comment]
@jorgeorpinel feel free to merge it you feel it's done!

@casperdcl good stuff! thanks!

**Comment by jorgeorpinel** [comment]
> how about speed this time?

Still a few minutes, maybe under 10 though. Drained 4% of my battery too haha. Seems reasonable to me. The tutorial runs this 3 times though.

**Pull request merged**